services/user.py

```diff
@@ -0,0 +1,49 @@
+from django.core.exceptions import ObjectDoesNotExist
+from django.db import transaction
+
+from db.models import User
+
+
+@transaction.atomic
+def create_user(username: str,
+                password: str,
+                email: str = None,
+                first_name: str = "",
+                last_name: str = "") -> User:
+    user = User.objects.create_user(
+        username=username,
+        password=password,
+        email=email,
+        first_name=first_name,
+        last_name=last_name
+    )
+    return user
+
+
+def get_user(user_id: int) -> User:
+    return User.objects.get(id=user_id)
```

Comment:
Consider adding error handling for the `User.objects.get(id=user_id)` call in the `get_user` function to manage cases where a user with the given ID does not exist. This can prevent the function from raising a `DoesNotExist` exception.